components/plugin_marketplace/marketplace_modal.scss

```diff
@@ -17,6 +17,23 @@
         padding-right: 15px;
     }
 
+    div.row {
+        margin-top: 10px;
+
+        div {
+            font-size: 14px;
+            text-align: center;
+            &:nth-child(2) {
```

Comment:
is this css for the `count` section? if so, i think better to have a dedicated class name for it as opposed to doing this, if someone decides to add more elements to that navigation row, the padding would break. If you have a dedicated name like `.count` then no matter where the count section is placed in the hierarchy, it wouldn't break padding.

Reply:
Done :+1: